extensions/objects/templates/image-with-regions-editor.component.ts

```diff
@@ -415,8 +416,12 @@ export class ImageWithRegionsEditorComponent implements OnInit {
   onSvgMouseMove(evt: MouseEvent): void {
     const svgElement: SVGSVGElement = this.el.nativeElement.querySelectorAll(
       '.oppia-image-with-regions-editor-svg')[0];
-    this.mouseX = evt.pageX - svgElement.getBoundingClientRect().left;
-    this.mouseY = evt.pageY - svgElement.getBoundingClientRect().top;
+    this.mouseX = evt.pageX - (
```

Comment:
Can you please add a comment with a brief explanation of the logic and link the docs as mentioned in the PR description?

Reply:
Done!